### ISSUE TITLE
Java 8 is not supported

### ISSUE THREAD

**Issue by momega** [nkonev/r2dbc-migrate#2]
Recently I have found your library and it seems to me that it will help with migrating my database in r2dbc environment. But I found than I cannot use it because your library is compilted with higher version of Java. 

<java.version>11</java.version>

Error message looks like this:

```
Caused by: java.lang.UnsupportedClassVersionError: name/nkonev/r2dbc/migrate/autoconfigure/R2dbcMigrateAutoConfiguration$R2dbcConnectionFactoryDependsOnBeanFactoryPostProcessor has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method) ~[na:1.8.0_242]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:757) ~[na:1.8.0_242]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[na:1.8.0_242]
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468) ~[na:1.8.0_242]
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74) ~[na:1.8.0_242]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369) ~[na:1.8.0_242]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363) ~[na:1.8.0_242]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_242]
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362) ~[na:1.8.0_242]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419) ~[na:1.8.0_242]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352) ~[na:1.8.0_242]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352) ~[na:1.8.0_242]
	at java.lang.Class.forName0(Native Method) ~[na:1.8.0_242]
	at java.lang.Class.forName(Class.java:348) ~[na:1.8.0_242]
	at org.springframework.util.ClassUtils.forName(ClassUtils.java:285) ~[spring-core-5.2.6.RELEASE.jar:5.2.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanDefinition.resolveBeanClass(AbstractBeanDefinition.java:469) ~[spring-beans-5.2.6.RELEASE.jar:5.2.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doResolveBeanClass(AbstractBeanFactory.java:1545) ~[spring-beans-5.2.6.RELEASE.jar:5.2.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.resolveBeanClass(AbstractBeanFactory.java:1472) ~[spring-beans-5.2.6.RELEASE.jar:5.2.6.RELEASE]
	... 16 common frames omitted
```

I still using Java 8. Will be possible to compile the library with target compatibility with java 8.
Thanks.

**Comment by nkonev** [comment]
I've just released 1.0.3, it will be available at the maven central after several hours.

**Comment by nkonev** [comment]
Already https://repo.maven.apache.org/maven2/name/nkonev/r2dbc-migrate/r2dbc-migrate-core/1.0.3/
